sqlglot/parser.py

```diff
@@ -1511,6 +1535,21 @@ def _parse_index(self) -> exp.Expression:
             columns=self._parse_expression(),
         )
 
+    def _parse_create_table_index(self):
+        unique = self._match_text_seq("UNIQUE")
+        primary = self._match_text_seq("PRIMARY")
+        self._match(TokenType.INDEX)
+        index = self._parse_id_var()
+        return self.expression(
+            exp.Index,
+            this=index,
+            columns=self.expression(
```

Comment:
i'm not sure we want tuples here, it should probably just be an array 

columns=self._parse_wrapped_csv(self._parse_column)

Reply:
Got it. Is this how I should handle it in the generator?

`ind_columns = f' ({", ".join([self.sql(col) for col in index.args.get("columns")])})'`

Reply:
> Got it. Is this how I should handle it in the generator?

You can use the `expressions` directly for this, I think. It defaults to `","` as a separator and you can provide `"columns"` as a key.